util/constants.py

```diff
@@ -3,6 +3,7 @@
 ZERO_ADDRESS = "0x0000000000000000000000000000000000000000"
 
 MARKET_ASSET_BASE_URL = "https://market.oceanprotocol.com/asset/"
```

Comment:
It appears that `MARKET_ASSET_BASE_URL` is unused? If yes, you can delete the line.

(It's not really this PR, however it's ok if we remove it here because it's kinda related)